From f07669df4c8df0b7134ae94be20a8b61bd157168 Mon Sep 17 00:00:00 2001
Message-Id: <f07669df4c8df0b7134ae94be20a8b61bd157168.1649662072.git.geliang.tang@suse.com>
From: Yang Yingliang <yangyingliang@huawei.com>
Date: Fri, 2 Apr 2021 09:26:34 +0800
Subject: [PATCH] libbpf: Remove redundant semi-colon
Git-commit: f07669df4c8df0b7134ae94be20a8b61bd157168
Patch-mainline: v5.13-rc1
References: bsn#127
Modified-by-SEL: No


Remove redundant semi-colon in finalize_btf_ext().

Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20210402012634.1965453-1-yangyingliang@huawei.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/lib/bpf/linker.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/lib/bpf/linker.c b/tools/lib/bpf/linker.c
index 46b16cbdcda3..4e08bc07e635 100644
--- a/tools/lib/bpf/linker.c
+++ b/tools/lib/bpf/linker.c
@@ -1895,7 +1895,7 @@ static int finalize_btf_ext(struct bpf_linker *linker)
 	hdr->func_info_len = funcs_sz;
 	hdr->line_info_off = funcs_sz;
 	hdr->line_info_len = lines_sz;
-	hdr->core_relo_off = funcs_sz + lines_sz;;
+	hdr->core_relo_off = funcs_sz + lines_sz;
 	hdr->core_relo_len = core_relos_sz;
 
 	if (funcs_sz) {
-- 
2.34.1

